src/components/search/Pagination.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from "react";
+import { SearchContext } from "../../contexts/SearchContext";
+import PageNumber from "./PageNumber";
+
+function Pagination() {
+  const { pagination } = useContext(SearchContext);
+  const pages = Array.from(
+    Array(pagination?.total_pages ?? 1),
+    (_, index) => index + 1,
+  );
```

Comment:
Javascript doesn't have a `range()` function like PHP, so we have to do this convoluted song and dance.

Reply:
I moved this into the `utils.js` file.